### DISCUSSION TITLE
Blank Space

### DISCUSSION THREAD

**Discussion by joaogabrieldasilva** [Shopify/flash-list#924] [Q&A]
I'm trying to render these cards using flashlist, but it keeps showing this blank space. I'm kind of new to recycling and I wanted to ask if I am doing something wrong here. I have already tried the following steps:

* Setting the correct estimatedItemSize prop
* use "memo"
* run in release mode
* add the recycling method with useRef but it didn't change anything, also, should I use this recycling method even if my card doesn't rely on any state?

Here I have my components

List component:

![image](https://github.com/Shopify/flash-list/assets/73562417/100bfe11-a1e1-4799-a03b-41a8bf5357cd)

Render Method
![image](https://github.com/Shopify/flash-list/assets/73562417/38385dad-e0f9-4690-8dac-cc29a73464c6)

Card
![image](https://github.com/Shopify/flash-list/assets/73562417/3ae705db-1e36-42e6-9e02-ae12670ad98e)

And heres is the evidence

[evidence.webm](https://github.com/Shopify/flash-list/assets/73562417/e7529501-abc1-4ffe-95c5-9c1e7e3a1d39)

**Comment by a1danw** [comment]
I am also struggling with the same issue and there doesn't seem to be any fix
